v2/tools/generator/internal/codegen/pipeline/report_resource_versions.go

```diff
@@ -76,7 +76,7 @@ func (r *ResourceVersionsReport) WriteTo(outputPath string) error {
 }
 
 // WriteToBuffer creates the report in the provided buffer
-func (r *ResourceVersionsReport) WriteToBuffer(buffer *strings.Builder) {
+func (r *ResourceVersionsReport) WriteToBuffer(buffer *strings.Builder, _ string) {
```

Comment:
Need to fix this code up, but for now will leave it so that can merge. Will send a PR to fix tomorrow.